.eslintrc.js

```diff
@@ -194,7 +195,7 @@ module.exports = {
         '@typescript-eslint/no-unsafe-call': 'off',
         '@typescript-eslint/no-unsafe-member-access': 'off',
         '@typescript-eslint/no-unsafe-return': 'off',
-        'eslint-plugin/no-identical-tests': 'error',
+        'eslint-plugin/consistent-output': 'off',
```

Comment:
I disabled [eslint-plugin/consistent-output](https://github.com/not-an-aardvark/eslint-plugin-eslint-plugin/blob/master/docs/rules/consistent-output.md) for two reasons:

1. There are 100 violations so we could follow-up to enable it in a separate PR if we want to.
2. I think it's less useful now that ESLint v7 [validates](https://eslint.org/docs/user-guide/migrating-to-7.0.0#rule-tester-strict) that test cases provide output when they produce an autofix, and it's tedious to include `output: null` when not necessary. It might be [removed](https://github.com/not-an-aardvark/eslint-plugin-eslint-plugin/pull/258#issuecomment-1179173504) as `recommended` eventually.

Reply:
Heh, yeah, now that ESLint is already at 8.21.0, it feels like removing from recommended is the right step 😄 

Could you just add a todo comment around this line with a link to an issue? Either one on this repo if you think we should enable it, or one on the eslint-plugin repo if you think it should be removed from `recommended`?

Reply:
Filed and added a link to this issue: https://github.com/not-an-aardvark/eslint-plugin-eslint-plugin/issues/284